Makefile

```diff
@@ -282,24 +282,24 @@ policies/glossary.html: $(GLOSSARY) bin/md-to-html5 Makefile
 newsflash.inc: news/newsflash.inc
 	@rm -f $@
 	head -7 $? >$@
-sitemap sitemap.txt:
+sitemap sitemap.txt: bin/mk-sitemap Makefile
 	@rm -f sitemap.txt
 	./bin/mk-sitemap master $(SERIES) > sitemap.txt
 
-community/committers.inc: $(PERSONDB)
+community/committers.inc: $(PERSONDB) bin/mk-committers Makefile
 	@rm -f $@
 	wget -q https://api.openssl.org/0/Group/commit/Members
 	./bin/mk-committers <Members >$@
 	@rm -f Members
 
-community/otc.inc: $(PERSONDB)
+community/otc.inc: $(PERSONDB) bin/mk-omc Makefile
 	./bin/mk-omc -n -p -t 'OTC Members' otc otc-inactive > $@
-community/omc.inc: $(PERSONDB)
+community/omc.inc: $(PERSONDB) bin/mk-omc Makefile
 	./bin/mk-omc -n -e -l -p -t 'OMC Members' omc omc-inactive > $@
-community/omc-alumni.inc: $(PERSONDB)
+community/omc-alumni.inc: $(PERSONDB) bin/mk-omc Makefile
 	./bin/mk-omc -n -l -t 'OMC Alumni' omc-alumni omc-emeritus > $@
 
-news/changelog.inc: news/changelog.txt bin/mk-changelog
+news/changelog.inc: news/changelog.txt bin/post-process-html5 Makefile
```

Comment:
Is losing bin/mk-changelog a problem here?

Reply:
If you look at the commands, you will see that it's not used anywhere any more.  We could actually drop `bin/mk-changelog` altogether, 'cause we don't use it any more...

Reply:
That happened in commit 4b0220368e888aab29972537aff8602a45b724e9